CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Added support to create and invoke stored procedures, UDFs and UDTFs with optional arguments.
 - Added support for column lineage in the DataFrame.lineage.trace API.
 - Added support for passing `INFER_SCHEMA` options to `DataFrameReader` via `INFER_SCHEMA_OPTIONS`.
+- Added support for passing `parameters` parameter to Column.rlike and Column.regexp.
```

Comment:
I don't see parameters support mentioned in the regexp function?
https://docs.snowflake.com/en/sql-reference/functions/regexp

Reply:
also we can update the local testing part for this new parameters support

Reply:
Do you mean to add a note in the local testing patch notes?

Reply:
yep